src/lib/parse/CompoundValueNode.cpp

```diff
@@ -818,201 +819,92 @@ std::string CompoundValueNode::render(ApiVersion apiVersion, bool noComma, bool
 
 /* ****************************************************************************
 *
-* toJson -
+* CompoundValueNode:toJson
 *
-* FIXME P3: isLastElement is not used and should be removed
 */
-std::string CompoundValueNode::toJson(bool isLastElement, bool comma)
+std::string CompoundValueNode::toJson(bool toplevel)
 {
-  std::string  out       = "";
-  bool         jsonComma = false;
-  std::string  key       = name;
+  std::string out;
 
-  if (container != NULL)
+  switch(valueType)
```

Comment:
Space missing. 'switch' is not a function ...

Reply:
Fixed in 295296b